functions/backup.bash

```diff
@@ -507,19 +507,24 @@ setup_mirror_SD() {
     if ! (whiptail --title "Copy internal SD to $dest" --yes-button "Continue" --no-button "Back" --yesno "$infoText" 12 116); then echo "CANCELED"; return 0; fi
   fi
 
-  if [[ "$destSize" -ge "$srcSize" ]]; then
-    mountUnit="$(basename "${storageDir}").mount"
-    systemctl stop "${mountUnit}"
-    # copy partition table
-    start="$(fdisk -l /dev/mmcblk0 | head -1 | cut -d' ' -f7)"
-    ((destSize-=start))
-    (sfdisk -d /dev/mmcblk0; echo "/dev/mmcblk0p3 : start=${start},size=${destSize}, type=83") | sfdisk --force "$dest"
-    partprobe
-    cond_redirect mke2fs -F -t ext4 "${dest}3"
-    if ! sed -e "s|%DEVICE|${dest}3|g" -e "s|%STORAGE|${storageDir}|g" "${BASEDIR:-/opt/openhabian}"/includes/storage.mount-template > "${serviceTargetDir}/${mountUnit}"; then echo "FAILED (create storage mount)"; fi
-    if ! cond_redirect chmod 644 "${serviceTargetDir}/${mountUnit}"; then echo "FAILED (permissions)"; return 1; fi
-    if ! cond_redirect systemctl -q daemon-reload &> /dev/null; then echo "FAILED (daemon-reload)"; return 1; fi
-    if ! cond_redirect systemctl enable --now "$mountUnit"; then echo "FAILED (enable storage mount)"; return 1; fi
+  mountUnit="$(basename "${storageDir}").mount"
+  systemctl stop "${mountUnit}"
+  if sfdisk -d ${src}$ | grep -q "^${src}p3"; then
```

Comment:
```suggestion
  if sfdisk -d ${src} | grep -q "^${src}p3"; then
```

Reply:
yes it was a copy'n paste typo that went unspotted